README.rst

```diff
@@ -93,7 +93,7 @@ Example Usage
     >>> local
     <Arrow [2013-05-11T13:23:58.970460-07:00]>
 
-    >>> local.timestamp()
+    >>> local.timestamp
```

Comment:
Arrow timestamp is a method. If I recall correctly, it was a property in the past.

Latest:
```
>>> import arrow
>>> arrow = arrow.Arrow(2015, 3, 12)
>>> arrow.timestamp
<bound method Arrow.timestamp of <Arrow [2015-03-12T00:00:00+00:00]>>
>>> arrow.timestamp()
1426118400.0
```